apps/portals/adknowledgeportal/src/App.scss

```diff
@@ -1,7 +1,7 @@
 // Load variable_overrides first, so correct values for variables will be applied to the rest of the styles
 @use './config/style/variable_overrides';
 
-@use '@sage-bionetworks/synapse-portal-framework/src/style/App';
+@use '@sage-bionetworks/synapse-portal-framework/style/App';
```

Comment:
The Vite config was updated to improve hot-module reloading & imports from the synapse-portal-framework package, so SCSS imports had to update to accomodate this.